include/CL/opencl.hpp

```diff
@@ -4524,6 +4524,13 @@ class Image : public Memory
         }
         return param;
     }
+
+    Buffer getBuffer()
```

Comment:
I'm not following why we need `getBuffer` - can you say more?

Any shared functionality between buffers and images should go in the "memory" class, so we shouldn't ever need to go from an "Image" to a "Buffer".

Reply:
I thought I needed a `cl::Buffer` to work with a `cl::KernelFunctor` in the blur example, but after your comment I see I can just use a `cl::Memory` with the same effect as well. Thanks for the comment. I've removed the `getBuffer` method.